stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -236,6 +238,36 @@ func (m *Manager) SoftDeleteTenantByID(ctx context.Context, tenantID string) (*T
 	return &t, nil
 }
 
+// DeactivateTenantDistributionAccount sets a distribution account of status ACTIVE to PENDING_USER_ACTIVATION for the given tenant id,
+// and is only used in the case where the distribution account is of type CircleDBVault.
+func (m *Manager) DeactivateTenantDistributionAccount(ctx context.Context, tenantID string) error {
+	updateQuery := `
+		UPDATE tenants t
+		SET
+			distribution_account_status = 'PENDING_USER_ACTIVATION'
+	`
+
+	queryParams := &QueryParams{
+		Filters: excludeInactiveTenantsFilters(),
+	}
+	queryParams.Filters[FilterKeyID] = tenantID
+	queryParams.Filters[FilterKeyDistributionAccountType] = schema.DistributionAccountCircleDBVault
+	queryParams.Filters[FilterKeyDistributionAccountStatus] = schema.AccountStatusActive
```

Comment:
I'm thinking we should explicitly return an error for when trying to deactivate an account that isn't a Circle account or active